testutil/testutil-core/src/main/java/com/sap/cloud/sdk/testutil/TenantContextApi.java

```diff
@@ -0,0 +1,29 @@
+package com.sap.cloud.sdk.testutil;
+
+import com.sap.cloud.sdk.cloudplatform.tenant.DefaultTenant;
+import com.sap.cloud.sdk.cloudplatform.tenant.Tenant;
+import com.sap.cloud.sdk.cloudplatform.tenant.TenantThreadContextListener;
+
+public interface TenantContextApi extends TestContextApi
+{
+    default Tenant setTenant()
+    {
+        return setTenant("tenant");
```

Comment:
[minor]

Let's make it a bit easier to search for this particular tenant id, just in case we are ever wondering where this tenant is coming from.

```suggestion
        return setTenant("default-test-tenant");
```